mantaray/views.py

```diff
@@ -542,6 +547,21 @@ def on_part(self, nick: str, reason: str | None) -> None:
             show_in_gui=self.server_view.should_show_join_leave_message(nick),
         )
 
+    def on_kick(self, channel: str, kicker_nick: str, kicked_nick: str, reason: str | None) -> None:
+        self.userlist.remove_user(kicked_nick)
+        if reason is None:
+            extra = ""
+        else:
+            extra = " " + reason + ""
+        self.add_message(
+            "*",
+            (kicker_nick, ["self-nick"]),
+            (' kicked ', []),
+            (kicked_nick, ["other-nick"]),
+            (" from the channel." + f"     (Reason: {extra})", []),
```

Comment:
The `Reason:` part is added here unconditionally, even if the reason is `None`.

Reply:
But, I think you should address my earlier review comment first: can the reason ever be None?

Reply:
For now I'm leaving it there, I've just changed the logic a little